urdf2webots/writeProto.py

```diff
@@ -56,7 +57,7 @@ def declaration(proto, robotName):
     """Prototype declaration."""
     proto.write('PROTO ' + robotName + ' [\n')
     proto.write('  field  SFVec3f     translation     0 0 0\n')
-    proto.write('  field  SFRotation  rotation        0 1 0 0\n')
+    proto.write('  field  SFRotation  rotation        ' + initRotation + '\n')
```

Comment:
I would rather pass this as an argument of the `declaration` function instead of a local variable.

Reply:
sure, go ahead. This was the quickest solution I came up with. A cleaner one is of course better

Reply:
I can do it too if you want

Reply:
As you prefeer, I can do it but tomorrow morning,

Reply:
done